samples/bookinfo/kube/bookinfo-details-v2.yaml

```diff
@@ -34,6 +34,6 @@ spec:
         ports:
         - containerPort: 9080
         env:
-        - name: WITH_ISTIO
+        - name: DO_NOT_USE_TLS_FOR_EXTERNAL_SERVICE_CALLS
```

Comment:
Shouldn’t we be setting this to false always to show SNI routing ? 
Also this name is too big. I suggest use_https_googleapis which is short n sweet

Reply:
@rshriram I propose to use this example to demonstrate both HTTP with TLS origination and HTTPS, and to compare between them. The idea is to demonstrate an organization that requires monitoring of the content and requires not to use TLS (HTTP with TLS origination). The environment variable simulates some general policy variable to enforce the policy of the organization. Maybe a simpler name would be `DO_NOT_USE_TLS` or `DO_NOT_ENCRYPT`.
Then I will compare this approach with HTTPS - an organization that does not have strict monitoring requirements and is OK to perform monitoring by SNI.